stl/inc/flat_map

```diff
@@ -960,210 +886,530 @@ private:
         _Guard._Clearable = nullptr;
     }
 
-    template <class _K>
-    _NODISCARD mapped_type& _At(const _K& _Key_val)
-        requires same_as<_K, key_type> || _Is_transparent_v<key_compare>
-    {
-        iterator _Position = find(_Key_val);
-        if (_Position == end()) {
-            _Xout_of_range("std::flat_map::at: the specified key does not exist.");
-        } else {
-            return _Position->second;
-        }
+    template <class _KeyTy>
+    size_type _Erase_key(_KeyTy&& _Key_val) {
+        const_iterator _Pos_begin = lower_bound(_STD forward<_KeyTy>(_Key_val));
+        const_iterator _Pos_end   = upper_bound(_STD forward<_KeyTy>(_Key_val));

```

Comment:
Silly question, would `equal_range` be faster? At least for the classic algorithm it is.